acceptance/bundle/deployment/bind/volume/script

```diff
@@ -0,0 +1,50 @@
+title "Bind volume test: "
+
+title "Substitute variables in the template: "
+BUNDLE_NAME_SUFFIX=$(uuid)
+export BUNDLE_NAME_SUFFIX
+
+VOLUME_NAME="volume-$(uuid)"
+SCHEMA_NAME="test-schema-$(uuid)"
+if [ -z "$CLOUD_ENV" ]; then
+    VOLUME_NAME="volume-6260d50f-e8ff-4905-8f28-812345678903"   # use hard-coded uuid when running locally
+    SCHEMA_NAME="test-schema-6260d50f-e8ff-4905-8f28-812345678903"
+fi
+export VOLUME_NAME
+export SCHEMA_NAME
+export CATALOG_NAME="main"
+envsubst < databricks.yml > out.yml && mv out.yml databricks.yml
+
+VOLUME_TYPE="MANAGED"
+
+title "Create a pre-defined schema (volume requires a schema): "
+$CLI schemas create "${SCHEMA_NAME}" ${CATALOG_NAME} | jq '{full_name, catalog_name}'
```

Comment:
Please 'trace' for such scripts, it's hard to read output.txt otherwise. The 'title' is only needed when it explains something not obvious from the command.